### PR TITLE
Update reporting_guidelines.md

### PR DESCRIPTION
**Pull request by andrina**
Updated to include the Reporting Tool as a means of reporting CoC violates. Updated language to reference stewards rather than community leaders

### CODE REVIEW

**Review by nrichers — APPROVED** [review]
Changes LGTM (looks GREAT to me), @andrina! 🚀 

One potential suggestion for the [Report a Message](https://yyjtech.slack.com/apps/A069TMQR4R1-report-a-message?tab=more_info) app link you included: that page might benefit from an usage example for the tool (`click on the “More actions” ellipsis (…) next to any message and select “Report message”`). I wasn't able to edit the description myself and it's outside the scope of this PR, but it might be helpful for folks looking for a practical answer.

File: reporting_guidelines.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Reporting Guidelines
 
 If you believe someone is violating the code of conduct (https://github.com/yyjtech/code-of-conduct) 
-we ask that you report it to the YYJ Tech community leaders here: https://goo.gl/forms/94uxzsNnwDLIotQ23
+we ask that you report it to the YYJ Tech community stewards through the Reporting Tool app within Slack. https://yyjtech.slack.com/apps/A069TMQR4R1-report-a-message?tab=more_info
```
Comment (jenreiher) [inline reporting_guidelines.md]: Can we please leave a screenshot with written instructions here as well?

File: reporting_guidelines.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ If you believe anyone is in physical danger, please notify appropriate law
 enforcement first. If you are unsure what law enforcement agency is appropriate,
 please include this in your report and we will attempt to notify them.
 
-If you included personal contact information, you will receive an email from the YYJ Tech community leaders acknowledging
+If you included personal contact information, you will receive communication from the YYJ Tech community stewards acknowledging
```
Comment (jenreiher) [inline reporting_guidelines.md]: can we link to `[community stewards](https://yyjtech.ca/channel-stewards/)` here?

File: reporting_guidelines.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ guarantee we’ll act on it.
 ### Appealing
 
 Only permanent resolutions (such as bans) may be appealed. To appeal a decision
-of the working group, contact community organizers with your appeal and we will review the case.
+of the working group, contact community stewards with your appeal and we will review the case.
```
Comment (jenreiher) [inline reporting_guidelines.md]: same here: `[community stewards](https://yyjtech.ca/channel-stewards/) `

**Comment by andrina** [comment]
> Changes LGTM (looks GREAT to me), @andrina! 🚀
> 
> One potential suggestion for the [Report a Message](https://yyjtech.slack.com/apps/A069TMQR4R1-report-a-message?tab=more_info) app link you included: that page might benefit from an usage example for the tool (`click on the “More actions” ellipsis (…) next to any message and select “Report message”`). I wasn't able to edit the description myself and it's outside the scope of this PR, but it might be helpful for folks looking for a practical answer.

Updated the Display Information for the App to reflect this.

File: reporting_guidelines.md (hunk 1 above)
Comment (andrina) [reply reporting_guidelines.md]: Updated this

File: reporting_guidelines.md (hunk 2 above)
Comment (andrina) [reply reporting_guidelines.md]: Added

File: reporting_guidelines.md (hunk 3 above)
Comment (andrina) [reply reporting_guidelines.md]: updated

**Pull request merged**